src/envs/screws.py

```diff
@@ -0,0 +1,501 @@
+"""Toy screw picking environment."""
+
+from typing import ClassVar, Dict, List, Optional, Sequence, Set, Tuple
+
+import matplotlib.pyplot as plt
+import numpy as np
+from gym.spaces import Box
+
+from predicators.src import utils
+from predicators.src.envs import BaseEnv
+from predicators.src.settings import CFG
+from predicators.src.structs import Action, Array, GroundAtom, Image, Object, \
+    ParameterizedOption, Predicate, State, Task, Type
+
+
+class ScrewsEnv(BaseEnv):
+    """Toy screw picking env.
+
+    In this environment, the agent controls a magnetic gripper that can
+    be used to pick up screws of different colors/sizes. The challennge
+    is for the agent to learn actions that don't precisely model all the
+    irrelevant screws that might be picked up when picking a desired
+    screw.
+    """
+    _gripper_width: ClassVar[float] = 0.5
+    _receptacle_width: ClassVar[float] = 0.6
+    _receptacle_height: ClassVar[float] = 0.2
+    _screw_width: ClassVar[float] = 0.05
+    _screw_height: ClassVar[float] = 0.05
+    _magnetic_field_dist: ClassVar[float] = 0.5
+    _num_screw_clusters: ClassVar[int] = 3
+    num_screws_train: ClassVar[List[int]] = CFG.screws_num_screws_train
```

Comment:
tests sometimes get screwy if we do this. suggest instead setting these fields that use `CFG` in `__init__()`

Reply:
(no pun intended)